src/hooks/useNamedPolicyPermissions/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import React, { useContext } from "react";
+import useSWR from "swr";
+import AccessControlContext from "../../contexts/accessControlContext";
+
+export default function useNamedPolicyPermissions(policyName) {
+  const { accessControl } = useContext(AccessControlContext);
+
+  async function getPermissions() {
```

Comment:
Why define it here instead of just having it as an anonymous function in the second parameter of `useSWR`?

Reply:
I remember we decided to do this when we were pair programming debugging why this hook got rerendered multiple times.

Reply:
Oh I thought you were referring to the accessControl line, I get confused with the lines GitHub decides to show in these things. Just because I'm not using it anywhere else but this hook, I didn't see a point in defining it elsewhere.